CommonTools/RecoAlgos/interface/RecoTrackSelectorBase.h

```diff
@@ -19,6 +19,8 @@ class RecoTrackSelectorBase {
     ptMin_(cfg.getParameter<double>("ptMin")),
     minRapidity_(cfg.getParameter<double>("minRapidity")),
     maxRapidity_(cfg.getParameter<double>("maxRapidity")),
+    meanPhi_((cfg.getParameter<double>("minPhi")+cfg.getParameter<double>("maxPhi"))/2.),
+    rangePhi_((cfg.getParameter<double>("maxPhi")-cfg.getParameter<double>("minPhi"))/2.),
```

Comment:
Mhm, no, my point was really to select tracks whose phi is around 0.
This
```
minPhi =  4.712    #  3/2 π
maxPhi =  1.571    #  1/2 π
```
should be equivalent to
```
minPhi = -1.571    #  -1/2 π == 3/2 π
maxPhi =  1.571    #  1/2 π
```